### PR TITLE
Added UUID to Person Edit Form which is read-only

### PR DESCRIPTION
**Pull request by fuatse**
Add the display of the UUID as requested in issue #2608

### CODE REVIEW

File: sormas-ui/src/main/java/de/symeda/sormas/ui/person/PersonEditForm.java [hunk 1]
```diff
@@ -83,6 +83,7 @@
 
 	private static final long serialVersionUID = -1L;
 
+	private static final String PERSON_UUID_LOC = "personUUIDLoc";
```
Comment (MateStrysewskeSym) [inline sormas-ui/src/main/java/de/symeda/sormas/ui/person/PersonEditForm.java:86]: Should be unneccessary

**Review by MateStrysewskeSym — CHANGES_REQUESTED** [review]
First of all, thanks for your submission and for supporting the project!

I'd prefer the UUID field to be read-only instead of not editable to make it appear as a label instead of a disabled text field. Have a look at the CaseDataForm to see how the case UUID is handled there. Also, please make sure to add an entry to the captions.properties file for Person.UUID and use "Person ID" as the value.

File: sormas-ui/src/main/java/de/symeda/sormas/ui/person/PersonEditForm.java (hunk 1 above)
Comment (fuatse) [reply sormas-ui/src/main/java/de/symeda/sormas/ui/person/PersonEditForm.java]: Very true.

**Review by MateStrysewske — APPROVED** [review]
Thank you for implementing the changes. Please make sure to reference the issue number in every commit next time you're contributing!

**Comment by fuatse** [comment]
> Thank you for implementing the changes. Please make sure to reference the issue number in every commit next time you're contributing!

Noted!

**Pull request merged**